.github/workflows/auto-label.yml

```diff
@@ -0,0 +1,64 @@
+name: Devtron-auto-labeller
+
+on:
+  issue_comment:
+    types: [created]
+
+
+jobs:
+  manage-labels:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Install GitHub CLI
```

Comment:
Might not need this because Github already gives gh cli pre installed in GH managed runners, try without this step

Reply:
Hi Pawan, thanks for poinitng it out, I'll remove this step, it is working on other repository.

Reply:
@pawan-mehta-dt I have removed and as @abhibhaw suggested that adding labels should be restricted to members of devtron, org membership verificaiton has been added.